falcon_multiqc/commands/query.py

```diff
@@ -0,0 +1,204 @@
+import click
+import subprocess
+import sys
+from database.crud import session_scope
+from database.models import *
+from sqlalchemy import Float
+from sqlalchemy.orm import load_only,Load
+from database.process_query import create_new_multiqc, create_csv
+
+# example command:
+# falcon_multiqc query --tool_metric --multiqc -d path
+# User will get prompted due to -tm flag, command will ask for which tool_metrics to filter on
+# examples
+# verifybamid AVG_DP < 28
+# picard_insertSize MEAN_INSERT_SIZE > 565
+# picard_wgsmetrics PCT_EXC_DUPE <= 0.04
+# Then hit 'enter'
+
+@click.command()
+@click.option('--select', is_flag=True, required=False, help = "Enter what you want to select on, e.g. select for sample_name, batch, etc. (deafult is sample_name)")
+@click.option('--tool_metric', is_flag=True, required=False, help = "Enter tool, metric, operator and number, e.g. verifybamid AVG_DP < 30")
+@click.option('--batch', is_flag=True, required=False, help = "Enter which batches to filter on e.g. AAA, BAA, etc.")
+@click.option('--cohort', is_flag=True, required=False, help = "Enter which cohort to filter on e.g. MGRB, cohort2, etc.")
+@click.option('--multiqc', is_flag=True, required=False, help = "Creates a multiqc report (user must select only for sample_name if so)")
+@click.option('--csv', is_flag=True, required=False, help = "Creates a csv report")
+@click.option("-d", "--directory", type=click.STRING, required=True, help="where query result will be saved")
+def cli(select, tool_metric, batch, cohort, multiqc, csv, directory):
+
```

Comment:
Also, here you need a comment with the tool's description that's showed in --help. Similar to save's.

Reply:
Alrighty, I didn't really know what we should write so I just gave a temporary one now